build.gradle

```diff
@@ -46,13 +45,14 @@ buildscript {
         maven { url "https://dl.bintray.com/kotlin/kotlin-dev" }
         maven { url "https://dl.bintray.com/jetbrains/markdown/" }
         maven { url "https://dl.bintray.com/arrow-kt/arrow-kt/" }
+        maven { url "https://oss.jfrog.org/artifactory/oss-snapshot-local/" }
     }
 
     dependencies {
         classpath "com.github.ben-manes:gradle-versions-plugin:$gradleVersionsPluginVersion"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-        classpath 'io.arrow-kt:ank-gradle-plugin:0.7.1'
+        classpath "io.arrow-kt:arrow-ank-gradle:0.10.5-SNAPSHOT"
```

Comment:
Hi @i-walker ! Just a comment about this change. 

With this change, the build is getting the Arrow Ank Gradle artifact with 0.10.5-SNAPSHOT version from the OSS repository, which is still using Kotlin 1.3.11.

Besides, this can work now because there is already a published artifact with that version in OSS repository. However, `VERSION_NAME` cannot be used as an alternative because it will fail during release time (the source code cannot depend on an artifact that is deployed from that source code).

Maybe that's the reason why that version is fixed from `0.7.1` because the Ank Gradle Plugin hasn't evolved since then. Some alternatives:

- Don't apply Arrow Ank Gradle Plugin. Add the `runAnk` task in `arrow-docs/build.gradle`.
- Exclude Arrow Ank Gradle Plugin. Change build and publish checks to control the order.

Reply:
;) I am aware of that- that is the same comment you made in the last PR. With the next PR it will use the one which is published now.

Reply:
> - Don't apply Arrow Ank Gradle Plugin. Add the runAnk task in arrow-docs/build.gradle

That suggestion doesn't make sense, because the only place runAnk is used is in the docs and it would lead to the same result.

> - Exclude Arrow Ank Gradle Plugin. Change build and publish checks to control the order.

The latter makes sense, but for this PR I solely focused on updating Ank. Maybe we can do that in another PR.

Reply:
Hi @rachelcarmena, I could not find something in the gradle docs, where we can set the build in a way where it uses the Gradle plugin from the project instead of a published one that has to exist in a repository. Do you know if there is an alternative? I am not very familiar with the current release cycle of arrow-ank-gradle.

Reply:
Hi! I'm looking at this task now! Let's see what we can do...

Reply:
Second alternative: #1925

Reply:
I discarded the first alternative because it implies duplicated code (between Gradle Plugin and the created tasks in Gradle file directly) and maintenance would be harder.

Reply:
Done, I merged #1925.